packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2287,17 +2287,25 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this.pendingStateManager.getLocalState();
     }
 
+    /**
+     * @returns true if summaries are explicitly disabled for this ContainerRuntime, false otherwise
+     */
+    public summariesDisabled(): boolean {
+        return this.runtimeOptions.summaryOptions.disableSummaries === true ||
+            this.runtimeOptions.summaryOptions.summaryConfigOverrides?.disableSummaries === true;
```

Comment:
Adding disableSummaries  to ISummaryConfiguration?

Reply:
Yes, it was added there in a separate change.